### PR TITLE
Update device_radix_sort.cuh

### PR DESCRIPTION
**Pull request by himanshu007-creator**
#282

### CODE REVIEW

File: cub/device/device_radix_sort.cuh [hunk 1]
```diff
@@ -66,7 +66,7 @@ namespace cub {
  * half-precision floating-point type.  Although the direct radix sorting
  * method can only be applied to unsigned integral types, DeviceRadixSort
  * is able to sort signed and floating-point types via simple bit-wise transformations
- * that ensure lexicographic key ordering. For floating-point types -0.0 and +0.0 are
+ * that ensure lexicographic key ordering. DeviceRadixSort is stable For floating-point types -0.0 and +0.0 are
```
Comment (allisonvacanti) [inline cub/device/device_radix_sort.cuh:69]: This needs to be blended into the text a bit more. It reads as "Device radix sort is stable for floating-point types", which is true, but the results are stable for non-floating-point types, too,

Can we switch this to "DeviceRadixSort is stable, and for floating-point types..." (or just make them separate sentences)?

**Review by allisonvacanti — CHANGES_REQUESTED** [review]

File: cub/device/device_radix_sort.cuh (hunk 1 above)
Comment (himanshu007-creator) [reply cub/device/device_radix_sort.cuh:69]: sure

**Review by allisonvacanti — APPROVED** [review]

**Comment by allisonvacanti** [comment]
Oops, I missed your update. This LGTM, I'll merge it soon -- thanks again!

BTW, don't feel obligated, but you may be interested in #289 :)

**Comment by allisonvacanti** [comment]
Replaced by PR #307, which adds the same info to the other radix sort algorithms.

**Pull request closed without merging**